Pods/Pods.xcodeproj/project.pbxproj

```diff
@@ -1014,7 +1013,7 @@
 			isa = PBXProject;
 			attributes = {
 				LastSwiftUpdateCheck = 1100;
-				LastUpgradeCheck = 1130;
+				LastUpgradeCheck = 1100;
```

Comment:
Are these changes from a pod install? any concern we went down?

Reply:
Good question. I think we don't need to commit the Pods project file in order to make the unit tests run, as I initially thought. Let me see if I can revert this change.

Reply:
The file has been reverted! Thanks Nicole!